components/job-orchestration/job_orchestration/scheduler/job_config.py

```diff
@@ -84,7 +84,8 @@ class SearchJobConfig(QueryJobConfig):
     network_address: typing.Optional[typing.Tuple[str, int]] = None
     aggregation_config: typing.Optional[AggregationConfig] = None
 
-    @validator("network_address")
+    @field_validator("network_address")
+    @classmethod
     def validate_network_address(cls, field):
```

Comment:
this is purely stylistic - we should change the second argument's name (`field`) to `value` to match the convention in the latest Pydantic docs

similarly, we should do the same for `valid_status` in `components/job-orchestration/job_orchestration/scheduler/scheduler_data.py`

@coderabbitai create an issue to track this